.github/workflows/chart-release-manual.yml

```diff
@@ -19,9 +24,11 @@ jobs:
       contents: read
   release:
     needs: scan
-    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/chart-release.yaml@1.3.1
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/chart-release.yaml@3.1.0
     secrets: inherit
     with:
       chart_name: dbildungs-iam
-      version_generation: specified
-      chart_version: ${{ inputs.chart_version }}
\ No newline at end of file
+      version_generation: specified 
+      tag_generation: specified 
+      chart_version: ${{ inputs.chart_version }}
+      image_tag: ${{ inputs.image_tag }}
```

Comment:
i think the names could be more concret so it is easier to understand 
version_generation:  helm_chart_version_generation: 
tag_generation:   image_tag_generation: